DiveLane/App/ViewControllerExtensions.swift

```diff
@@ -10,47 +10,48 @@ import UIKit
 
 extension UIViewController {
     func hideKeyboardWhenTappedAround() {
-        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(UIViewController.dismissKeyboard))
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self,
+                                                                 action: #selector(UIViewController.dismissKeyboard))
         tap.cancelsTouchesInView = false
         view.addGestureRecognizer(tap)
     }
-    
+
     @objc func dismissKeyboard() {
         view.endEditing(true)
     }
-    
+
     func setupViewResizerOnKeyboardShown() {
         NotificationCenter.default.addObserver(self,
-                                               selector: #selector(UIViewController.keyboardWillShowForResizing),
-                                               name: Notification.Name.UIKeyboardWillShow,
-                                               object: nil)
+                selector: #selector(UIViewController.keyboardWillShowForResizing),
+                name: Notification.Name.UIKeyboardWillShow,
+                object: nil)
         NotificationCenter.default.addObserver(self,
-                                               selector: #selector(UIViewController.keyboardWillHideForResizing),
-                                               name: Notification.Name.UIKeyboardWillHide,
-                                               object: nil)
+                selector: #selector(UIViewController.keyboardWillHideForResizing),
+                name: Notification.Name.UIKeyboardWillHide,
+                object: nil)
     }
-    
+
     @objc func keyboardWillShowForResizing(notification: Notification) {
         if let keyboardSize = (notification.userInfo?[UIKeyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue,
-            let window = self.view.window?.frame {
+           let window = self.view.window?.frame {
             // We're not just minusing the kb height from the view height because
             // the view could already have been resized for the keyboard before
             self.view.frame = CGRect(x: self.view.frame.origin.x,
-                                     y: self.view.frame.origin.y,
-                                     width: self.view.frame.width,
-                                     height: window.origin.y + window.height - keyboardSize.height)
+                    y: self.view.frame.origin.y,
```

Comment:
ssems, that is autocorrection, but it still a lot of corrections, by running `swiftlint autocorrect` - how it posssible? did you run linter for specific files?